rules/naming/src/PropertyRenamer.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Naming;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\PropertyFetch;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\VarLikeIdentifier;
+use Rector\Core\PhpParser\NodeTraverser\CallableNodeTraverser;
+use Rector\Core\Rector\AbstractRector\NameResolverTrait;
+use Rector\Core\Rector\AbstractRector\NodeTypeResolverTrait;
+use Rector\Naming\Guard\BreakingVariableRenameGuard;
+use Rector\Naming\ValueObject\PropertyRename;
+
+final class PropertyRenamer
+{
+    use NameResolverTrait;
+    use NodeTypeResolverTrait;
```

Comment:
These should be used only in `abstract class AbstractRector`.
Anywhere else DI should be used

Reply:
Yeah, that is something I want to ask about, one of them is not even used btw...
This because of `$this->isLocalPropertyFetchNamed`, it is present only in trait and it seems like there is no something similar in `NodeNameResolver`.

Reply:
What about move `isLocalPropertyFetchNamed()` to `NodeNameResolver`?

Reply:
Is it okay in your opinion to move this method to `NodeNameResolver`?

Reply:
Yes. Basically traits should only delegate, never make their own method. It creates crap design.

That was just my laziness and absence of PHPStan rule :)

Reply:
Done